.travis.yml

```diff
@@ -2,19 +2,24 @@ language: c
 
 sudo: required
 
-os:
-  - linux
-  - osx
-
-compiler:
-  - gcc
-  - clang
+matrix:
+  include:
+    - os: linux
+      compiler: gcc
+    - os: linux
+      compiler: clang
+    - os: osx
+      compiler: gcc
+    - os: osx
+      compiler: clang
+    - os: linux
+      env: ASTYLE_FORMAT=true
```

Comment:
maybe move this build to be the first in the list
it seems Travis-CI takes them in top-to-bottom order, and it would be neat if this runs first